src/endpoints/utils.ts

```diff
@@ -0,0 +1,20 @@
+export function buildQueryParams(
+    params?: {[key: string]: any},
```

Comment:
Can we use `unknown` here instead of `any`?

Reply:
What's the advantage of using `unknown`.